query-result-writer.hh

```diff
@@ -135,6 +138,19 @@ public:
 
     result_memory_accounter& memory_accounter() { return _memory_accounter; }
 
+    stop_iteration bump_and_check_tombstone_limit() {
+        ++_tombstones;
+        if (!_tombstone_limit || _tombstones < _tombstone_limit) {
```

Comment:
Avi is right that if the default tombstone_limit was maxint, this if() would have been simpler.